.github/workflows/client-ios.yml

```
@@ -42,14 +42,14 @@ concurrency:
 
 jobs:
   build:
-    runs-on: macos-11
+    runs-on: macos-12
     steps:
       - name: 👀 Checkout
         uses: actions/checkout@v3
         with:
           submodules: true
-      - name: 🔨 Switch to Xcode 13.2.1
-        run: sudo xcode-select --switch /Applications/Xcode_13.2.1.app
+      - name: 🔨 Switch to Xcode 14.0
+        run: sudo xcode-select --switch /Applications/Xcode_14.0.app
```

Why not use the latest available 14.1?
```suggestion
        run: sudo xcode-select --switch /Applications/Xcode_14.1.app
```

---

It wasn't clear if GitHub supported 14.1 yet but I will try that next.

---

They do, see https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md

---

Done